.php-cs-fixer.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+declare(strict_types=1);
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+$rules = [
+    'array_syntax' => ['syntax' => 'short'],
+    'binary_operator_spaces' => [
```

Comment:
What does this config vs `['operators' => ['=' => 'align', '=>' => 'align']]` look like?